src/main/java/no/nav/arbeidsgiver/sykefravarsstatistikk/api/importering/autoimport/SykefraværsstatistikkImporteringService.java

```diff
@@ -56,6 +62,15 @@ public Importeringstatus importerHvisDetFinnesNyStatistikk() {
         statistikkRepository.hentSisteÅrstallOgKvartalForSykefraværsstatistikk(
             Statistikkilde.VIRKSOMHET)
     );
+
+    if (Arrays.asList(environment.getActiveProfiles()).contains("dev")
```

Comment:
Jeg har lyst til å gjøre det sjekken om. 
Litt bekymret over at `default` er å generere data. 
Om sjekken filer i `dev-test` for en eller annen grunn, vil data bli generert i `prod`, og det ønsker vi ikke.